appuselfbot.py

```diff
@@ -106,24 +109,25 @@ def Wizard():
         Wizard()
 
 shutdown = False
-if not get_config_value('config', 'run_as_superuser'):
+if os.name == 'nt':
+    try:
+        # only windows users with admin privileges can read the C:\windows\temp
+        temp = os.listdir(os.sep.join([os.environ.get('SystemRoot','C:\\windows'),'temp']))
+    except:
+        shutdown = False
+    else:
+        shutdown = True
+else:
+    if 'SUDO_USER' in os.environ and os.geteuid() == 0:

```

Comment:
Separately confirmed that running the script as root, while logged in directly as root, will bypass this check (`sudo su` counts as sudo usage and is not true "logged in as root"). I am going to make a commit to this branch directly so that we can push this interim fix out asap while the dev team mulls over whether we should still include this check in the `rewrite` branch.